charts/rancher-csp-billing-adapter/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: rancher-csp-billing-adapter
+description: Cloud Service Provider Marketplace Billing Adapter for Rancher.
+version: 999
+appVersion: 999
```

Comment:
are these version numbers typical?

Reply:
The CRD is generated by kubebuilder so I don't think these versions are significant from our standpoint.